pkg/charts/additionalchart.go

```diff
@@ -47,6 +47,9 @@ func (c *AdditionalChart) ApplyMainChanges(pkgFs billy.Filesystem) error {
 	if err := helm.CopyCRDsFromChart(pkgFs, mainChartWorkingDir, path.ChartCRDDir, c.WorkingDir, c.CRDChartOptions.CRDDirectory); err != nil {
 		return fmt.Errorf("Encountered error while trying to copy CRDs from %s to %s: %s", mainChartWorkingDir, c.WorkingDir, err)
 	}
+	if err := filesystem.ArchiveDir(pkgFs, c.WorkingDir, c.CRDChartOptions.CRDDirectory, filepath.Join(c.WorkingDir, path.ChartExtraFileDir)); err != nil {
```

Comment:
> Either we should ArchiveDir or CopyCRDsFromChart, is there any reason why a CRD chart should need both?

`CopyCRDsFromChart` is kept because the rancher-monitoring-crd chart needs those crd files to render its final charts. Keeping them makes the chance on both here and the monitoring chart relatively easy and small.